sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -793,15 +789,16 @@ def _from_certificate_policy_bundle(cls, certificate_policy_bundle):
             ),
             lifetime_actions=lifetime_actions,
             exportable=key_properties.exportable if key_properties else None,
-            key_type=KeyType(key_properties.key_type) if key_properties and key_properties else None,
+            key_type=KeyType(key_properties.key_type) if key_properties and key_properties.key_type else None,
             key_size=key_properties.key_size if key_properties else None,
             reuse_key=key_properties.reuse_key if key_properties else None,
-            curve=KeyCurveName(key_properties) if key_properties and key_properties.curve else None,
+            curve=KeyCurveName(key_properties.curve) if key_properties and key_properties.curve else None,
```

Comment:
This looks like something that should have failed a test, do we have coverage here?

Reply:
I don't think we actually have a certificate policy that uses a curve, i'll add that to one of them

Reply:
I'm going to opening up a different pr with increased test coverage for certificate policy